src/main/java/net/testudobank/MvcController.java

```diff
@@ -308,7 +312,7 @@ public String submitWithdraw(@ModelAttribute("user") User user) {
     int userOverdraftBalanceInPennies = TestudoBankRepository.getCustomerOverdraftBalanceInPennies(jdbcTemplate, userID);
     if (userWithdrawAmtInPennies > userBalanceInPennies) { // if withdraw amount exceeds main balance, withdraw into overdraft with interest fee
       int excessWithdrawAmtInPennies = userWithdrawAmtInPennies - userBalanceInPennies;
-      int newOverdraftIncreaseAmtAfterInterestInPennies = (int)(excessWithdrawAmtInPennies * INTEREST_RATE);
+      int newOverdraftIncreaseAmtAfterInterestInPennies = (int)applyInterestRateToPennyAmount(excessWithdrawAmtInPennies);
```

Comment:
It would've been better to make the helper method return an `int` and apply the `(int)` cast within the method itself. This will save future developers the headache of making sure to apply the `(int)` cast in-line.